CHANGELOG.md

```diff
@@ -63,7 +63,7 @@ A big thanks to the 3 contributors who made this release possible. Here are some
 
 ## 5.0.0
 
-_Nov 16, 2021_
+_Nov 23, 2021_
 
 🎉 We are execited to introduce [MUI X v5.0.0](https://mui.com/blog/mui-x-v5/) 🎉!
```

Comment:
You need to merge with `master` first. This typo was already fixed: https://github.com/mui-org/material-ui-x/blob/e126eb9179c2643fb2bc98619909273e54bdccc5/CHANGELOG.md?plain=1#L68